### PR TITLE
Internal: add color GraphicalElements4

### PR DESCRIPTION
**Pull request by SatseAndrew**
# Why?

There is currently an issue in discover where this happens in dark mode.

Screenshot 2022-05-11 at 10 26 05

# What?

Add new color graphicalElements4.

# UI Changes

This is how the icon will look with the new color.

Screenshot 2022-05-11 at 10 29 22

### CODE REVIEW

**Review by kentrh — CHANGES_REQUESTED** [review]
Test are failing in `ColorTests` because the color hasn't been added there, also a demo of the color is missing in `ColorDemoView`

File: DemoApp/DemoApp/Views/DNA/ColorDemoView.swift [hunk 1]
```diff
@@ -153,6 +153,7 @@ struct ColorDemoView: View {
                 ColorDemo(name: "graphicalElements1", color: .graphicalElements1),
                 ColorDemo(name: "graphicalElements2", color: .graphicalElements2),
                 ColorDemo(name: "graphicalElements3", color: .graphicalElements3),
+                ColorDemo(name: "graphicalElements4", color: .graphicalElements3),
```
Comment (kentrh) [inline DemoApp/DemoApp/Views/DNA/ColorDemoView.swift:156]:
```suggestion
                ColorDemo(name: "graphicalElements4", color: .graphicalElements4),
```

**Comment by fespinoza** [comment]
I added a task for us to take a look at the CI setup 😨

**Review by kentrh — APPROVED** [review]
👍

**Pull request merged**